features/db-size.feature

```diff
@@ -202,3 +202,62 @@ Feature: Display database size
       """
       KB
       """
+
+  Scenario: Display ordered table sizes for a WordPress install
+    Given a WP install
+
+    When I run `wp db size --tables --order=DESC`
+    Then STDOUT should contain:
+      """
+      wp_users	65536 B
+      wp_usermeta	49152 B
+      wp_terms	49152 B
+      wp_termmeta	49152 B
+      wp_term_taxonomy	49152 B
+      wp_term_relationships	32768 B
+      wp_posts	81920 B
+      wp_postmeta	49152 B
+      wp_options	49152 B
+      wp_links	32768 B
+      wp_comments	98304 B
+      wp_commentmeta	49152 B
+      """
+
+    When I run `wp db size --tables --orderby=size`
+    Then STDOUT should contain:
+      """
+      wp_links	32768 B
+      wp_term_relationships	32768 B
+      wp_commentmeta	49152 B
+      wp_options	49152 B
+      wp_postmeta	49152 B
+      wp_term_taxonomy	49152 B
+      wp_termmeta	49152 B
+      wp_terms	49152 B
+      wp_usermeta	49152 B
+      wp_users	65536 B
+      wp_posts	81920 B
+      wp_comments	98304 B
+      """
```

Comment:
If WP will ever add/change even 1 option, these numbers will change. And that could happen with every table.
And the order itself might even change.
In all other current tests there is no check with absolute numbers.

My suggestion is after installing, deleting all (sample) posts/post_meta,
Adding a couple of sample posts and postmeta. 
That way the numbers should always be the same.

Reply:
> My suggestion is after installing, deleting all (sample) posts/post_meta,
> Adding a couple of sample posts and postmeta.
> That way the numbers should always be the same.

I think I'd go one step further: don't include absolute numbers at all. I suspect there will still be variables we can't control, which will cause the tests to fail intermittently.

Instead, let's create some contrived scenario where one table obviously has more data than another, and then verify the expected behavior with our `--order=<order>` argument.

Reply:
I had originally pulled the hard-coded sizes based on another of the `size.feature` tests:
```feature
    When I run `wp db size --tables`
    Then STDOUT should contain:
      """
      wp_posts	81920 B
      """
```

I'll look into how this could be stabilized without relying on hard-coded values and specific orders.

Reply:
@kjohnson Oh! You could read the current table size to an environment variable, and then use that in your test.

Reply:
I figured out that if I switch to a JSON output format, then I can more easily determine the first row based on the JSON syntax.

```feature
  Scenario: Display ordered table sizes for a WordPress install
    Given a WP install
    And I run `wp post generate --post_type=page --post_status=draft --count=300`

    When I run `wp db size --tables --order=desc --orderby=size --format=json`
    Then STDOUT should contain:
      """
      [{"Name":"wp_posts",
      """
```